kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -28,10 +32,44 @@
 
   module.exports = {
     name: 'channelList',
-    $trNameSpace: 'channelList',
+    $trNameSpace: 'coachRecentPageChannelList',
     $trs: {
       channels: 'Channels',
       channelList: 'Channel list',
+      lastActive: 'Last active',
+      daysPassed: 'X day(s) ago',
+      monthsPassed: 'X month(s) ago',
+      timePassed:
```

Comment:
Constructing this string a great opportunity for learning, but I think it would be nice if we had an implementation of https://github.com/yahoo/react-intl/wiki/Components#formattedrelative

@rtibbles what do you think?